src/services/helper/mentors.js

```diff
@@ -12,6 +12,7 @@ const apiEndpoints = require('@constants/endpoints')
 const apiBaseUrl = process.env.USER_SERIVCE_HOST + process.env.USER_SERIVCE_BASE_URL
 const request = require('@generics/requests')
 const sessionAttendees = require('@db/sessionAttendees/queries')
+const { RedisHelper } = require('@ankitpws/caching-library')
```

Comment:
@ankitpws using config we should be able to set the cache expiry time separately for redies and internal cache